document-generation/smartdocuments/src/main/kotlin/com/ritense/smartdocuments/web/rest/SmartDocumentsDemoResource.kt

```diff
@@ -50,15 +48,13 @@ class SmartDocumentsDemoResource(
         val smartDocumentsPlugin = pluginService.createPluginInstance(pluginConfigurationId) as SmartDocumentsPlugin
         val variables = runtimeService.getVariables(processInstanceId)
         val delegateExecutionSmall = DelegateExecutionSmall(processInstanceId, variables)
-        val properties = Mapper.INSTANCE.get().writeValueAsString(
-            SmartDocumentsPluginGenerateDocumentProperties(
-                templateGroup,
-                templateName,
-                DocumentFormatOption.valueOf(format),
-                templatePlaceholders
-            )
+        smartDocumentsPlugin.generate(
```

Comment:
I already made some changes that will conflict with this change...

Reply:
I think the demo endpoint is no longer needed. Since a plugin can now be called from the process